app/models/site_config.rb

```diff
@@ -73,7 +73,10 @@ class SiteConfig < RailsSettings::Base
   field :logo_svg, type: :string, default: ""
   field :primary_sticker_image_url, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/rainbowdev.svg"
 
-  # Rate Limits
+  # Monetization
+  field :payment_pointer, type: :string, default: "$ilp.uphold.com/24HhrUGG7ekn" # Experimental
```

Comment:
What is this default? Is it a special token?

Reply:
This is _our_ pointer. I just figured this is an okay default for now. Won't really make a big difference one way or another.